testsuite/tests/linear/should_fail/LinearPartialSig.hs

```diff
@@ -0,0 +1,5 @@
+{-# LANGUAGE LinearTypes #-}
+module LinearPartialSig where
+
+f :: a -->.(_) a
+f x = x
```

Comment:
What are we testing in this test? Can we document?

Reply:
Before ef4fa3397d, this caused an internal error. I added a one line description to the .hs file about the expected result but the motivation is that ghc shouldn't panic.